CRM/Core/Form/Search.php

```diff
@@ -204,6 +207,15 @@ protected function getEntityDefaults($entity) {
         if ($value !== FALSE) {
           $defaults[$fieldSpec['name']] = $value;
         }
+        if ($fieldSpec['type'] === CRM_Utils_Type::T_DATE || ($fieldSpec['type'] === CRM_Utils_Type::T_DATE + CRM_Utils_Type::T_TIME)) {
+          $low = CRM_Utils_Request::retrieveValue($fieldSpec['name'] . '_low', 'Timestampe', FALSE, NULL, 'GET');
```

Comment:
@eileenmcnaughton I think we should also include '_from'/'_to' names?

Reply:
Since we'll be converting the fields one by one hopefully we can just standardise them in the process.